src/ILCompiler.TypeSystem/tests/VirtualMethodEnumerationAlgorithmTests.cs

```diff
@@ -64,5 +64,20 @@ public void TestCachingEnumerationAlgorithm()
             Assert.Equal("ToString", testTypeMethods[0].Name);
             Assert.Equal(_testType, testTypeMethods[0].OwningType);
         }
+
+        [Fact]
+        public void TestFindBaseUnificationGroup()
```

Comment:
The other virtual function overriding tests are in `VirtualFunctionOverrideTests.cs`. The file you're modifying is testing odd functionalities I only added because David asked me to.